moveit_setup_assistant/moveit_setup_controllers/include/moveit_setup_controllers/moveit_controllers_config.hpp

```diff
@@ -100,59 +99,23 @@ class ControllersConfig : public SetupConfig
 
     std::string getDescription() const override
     {
-      return "Creates configurations for ros_controllers.";
+      return "Creates configurations for moveit_controllers.";
     }
 
     bool writeYaml(YAML::Emitter& emitter) override;
 
   protected:
-    ControllersConfig& parent_;
-  };
-
-  class GeneratedContollerLaunch : public TemplatedGeneratedFile
-  {
-  public:
-    GeneratedContollerLaunch(const std::filesystem::path& package_path, const GeneratedTime& last_gen_time,
-                             ControllersConfig& parent)
-      : TemplatedGeneratedFile(package_path, last_gen_time), parent_(parent)
-    {
-    }
-
-    bool hasChanges() const override
-    {
-      return parent_.changed_ || parent_.hasChangedGroups();
-    }
-
-    std::filesystem::path getRelativePath() const override
-    {
-      return "launch/ros_controllers.launch";
-    }
-
-    std::filesystem::path getTemplatePath() const override
-    {
-      return getSharePath("moveit_setup_controllers") / "templates" / getRelativePath();
-    }
-
-    std::string getDescription() const override
-    {
-      return "ros_controllers launch file";
-    }
-
-  protected:
-    ControllersConfig& parent_;
+    MoveItControllersConfig& parent_;
   };
 
   void collectFiles(const std::filesystem::path& package_path, const GeneratedTime& last_gen_time,
                     std::vector<GeneratedFilePtr>& files) override
   {
     files.push_back(std::make_shared<GeneratedControllersConfig>(package_path, last_gen_time, *this));
-    files.push_back(std::make_shared<GeneratedContollerLaunch>(package_path, last_gen_time, *this));
   }
 
-  void collectVariables(std::vector<TemplateVariable>& variables) override;
-
   /**
-   * \brief Adds a controller to ros_controllers_config_ vector
+   * \brief Adds a controller to controllers_config_ vector
```

Comment:
It would be a nice improvement if every function followed this documentation style with brief, then parameters (input or output, hopefully never output or mixed) and return, when applicable.